assets/sass/scaffolding/grid.scss

```diff
@@ -212,3 +212,43 @@
         margin: 0 !important;
     }
 }
+
+$num_columns: 12;
+$num_columns_mobile: 3;
+
+@mixin generate-column($className, $width) {
+    #{$className} {
+        flex-basis: $width;
+        flex-grow: 0;
+
+        .grid--padded & {
+            flex-basis: calc(#{$width} - #{$paddedGridGutter});
+        }
+
+        .grid--padded--large & {
+            flex-basis: calc(#{$width} - #{$paddedGridGutterLarge});
+        }
+    }
+}
+
+.row {
```

Comment:
🤓  Pedantic, but maybe `.grid-row` to make it explicit it's part of the grid classes?